core/server/common/src/main/java/alluxio/underfs/AbstractUfsManager.java

```diff
@@ -209,6 +210,30 @@ public UfsClient getRoot() {
     }
   }
 
+  @Override
+  public UfsClient getJournal() {
+    synchronized (this) {
+      if (mJournalUfsClient == null) {
+        String uri = ServerConfiguration.get(PropertyKey.MASTER_MOUNT_TABLE_JOURNAL_UFS);
+        boolean readOnly =
+            ServerConfiguration.getBoolean(PropertyKey.MASTER_MOUNT_TABLE_JOURNAL_READONLY);
+        boolean shared = ServerConfiguration
+            .getBoolean(PropertyKey.MASTER_MOUNT_TABLE_JOURNAL_SHARED);
+        Map<String, String> conf =
+            ServerConfiguration.getNestedProperties(PropertyKey.MASTER_MOUNT_TABLE_JOURNAL_OPTION);
+        addMount(IdUtils.JOURNAL_MOUNT_ID, new AlluxioURI(uri),
```

Comment:
It looks like we are reusing the mounts to store the journal UFS as well. Can you document that in a comment somewhere? Maybe near the top of this class?

Reply:
Done. added comments on the class java doc.